op-node/rollup/derive/data_source.go

```diff
@@ -110,15 +110,12 @@ func isValidBatchTx(tx *types.Transaction, receipt *types.Receipt, l1Signer type
 	if to == nil || *to != batchInboxAddr {
 		return false
 	}
-	seqDataSubmitter, err := l1Signer.Sender(tx) // optimization: only derive sender if To is correct
-	if err != nil {
-		logger.Warn("tx in inbox with invalid signature", "hash", tx.Hash(), "err", err)
-		return false
-	}
-	// some random L1 user might have sent a transaction to our batch inbox, ignore them
-	if seqDataSubmitter != batcherAddr {
-		logger.Warn("tx in inbox with unauthorized submitter", "addr", seqDataSubmitter, "hash", tx.Hash(), "err", err)
-		return false
-	}
+
+	// NOTE: contrary to a standard OP batcher, we can safely skip any verification related
+	// to the sender of the transaction. Indeed the Batch Inbox contract takes care of
+	// ensuring the sender of the batch information is a legitimate batcher.
```

Comment:
Would this cause inconsistency issues for the Caff node because it won't know whether a transaction is from an invalid sender in this case?